panther_analysis_tool/detection.py

```diff
@@ -147,6 +147,28 @@ def detection_evaluation_failed(self) -> bool:
         """Returns whether the detection function raises an error or an import error occurred"""
         return bool(self.detection_exception or self.setup_exception)
 
+    def ignore_errors(self, ignore_exception_types: Optional[List[Type[Exception]]] = None) -> None:
```

Comment:
Sorry I missed that in the previous review, it would be better if the parameter is not optional now, especially since this function is never called without parameters or with `None`.